### PR TITLE
Custodial account UI

### PR DESCRIPTION
**Pull request by itsrachelfish**
This PR updates the custom session UI to include custodial accounts. 

image

Depends on #451

### CODE REVIEW

File: app/src/store/views/addSessionView.ts [hunk 1]
```diff
@@ -203,19 +216,32 @@ export default class AddSessionView {
...
     let label = this.label;
+    let accountId = '';
 
     // Automatically generate human friendly labels for custom sessions
     if (label === '') {
       label = `My ${this.permissionType} session`;
     }
 
+    if (this.permissionType === 'custodial') {
+      const custodialAccount = await this.registerCustodialAccount();
```
Comment (jamaljsr) [inline app/src/store/views/addSessionView.ts:227]: nit: rename to `custodialAccountId` since the value is an ID, not an Account.

**Review by jamaljsr — COMMENTED** [review]
Overall, this looks great. I also saw the issue @guggero mentioned.

I'll reiterate here the suggestions I made over chat last week.

1. Keep the QR code icon enabled for custodial sessions
2. Add a custodial flag to the encoded data used in the TW url. Just be mindful of backwards compatibility for apps that may already use the existing QR code, like Zeus.

**Review by jamaljsr — APPROVED** [review]
tACK LGTM 👍

All clear from me!

**Review by guggero — APPROVED** [review]
LGTM :tada:

**Comment by itsrachelfish** [comment]
Had to rebase this branch after updating the `custom-connect-ui` branch with the latest changes from master

**Review by hexafluoride — APPROVED** [review]

**Comment by guggero** [comment]
Can we merge this into the `custom-connect-ui` branch and then merge https://github.com/lightninglabs/lightning-terminal/pull/451 as a whole? Looks like we got a bunch of reviews already (or do we want to wait for @dstrukt's review?).

**Comment by jamaljsr** [comment]
Agreed @guggero

**Pull request merged**